bin/coreos-install

```diff
@@ -380,11 +360,35 @@ do
         f) IMAGE_FILE="$OPTARG" ;;
         n) COPY_NET=1;;
         v) set -x ;;
-        h) echo "$USAGE"; exit;;
+        h) usage=1;;
         *) exit 1;;
     esac
 done
 
+if [[ -n ${usage} ]]; then
+    echo "\
+Usage: $0 -d <device> [options]
+Options:
+    -d DEVICE   Install Container Linux to the given device.
+    -V VERSION  Version to install (e.g. current) [default: ${VERSION_ID}].
```

Comment:
The word `default` is now actively misleading.  @dm0- suggested `using:` as an alternative, but I'm wondering whether it'd be better to a) leave the usage message as-is and b) have a dry-run option that checked invariants (the code below), printed its intentions, and exited.

Reply:
My opinion is that if we're going to have only one message we should switch to `using:`. If we do end up with two we should alter the original message to use the actual defaults rather than the current implementation.